zenoh/src/net/routing/dispatcher/pubsub.rs

```diff
@@ -173,7 +173,7 @@ pub(crate) fn undeclare_subscription(
         Resource::clean(&mut res);
         drop(wtables);
     } else {
-        tracing::error!("{} Undeclare unknown subscriber {}", face, id);
+        tracing::trace!("{} Undeclare unknown subscriber {}", face, id);
```

Comment:
Should we set to debug rather than trace ?

Reply:
I think setting it to Debug would send people in the wrong direction when debugging, since it's expected behavior when ACL is enabled, but the message sounds like something went wrong.

Reply:
Fine.
Maybe add a comment here stating that this is expected in case of ACL.